.github/workflows/ci.yml

```diff
@@ -207,35 +206,41 @@ jobs:
           conda info
           conda list
       - name: Install HDF5
-        if: matrix.part == 3
         run: sudo apt update && sudo apt install -y libhdf5-dev
       - shell: bash -l {0}
-        run: pytest modin/pandas/test/dataframe/test_${{matrix.part}}.py
-        if: matrix.part != 3
+        run: pytest modin/pandas/test/dataframe/test_binary.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_default.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_indexing.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_iter.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_join_sort.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_map_metadata.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_reduction.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_udf.py
+      - shell: bash -l {0}
+        run: pytest modin/pandas/test/dataframe/test_window.py
```

Comment:
since we have all that tests in a single GH action, maybe it would be easier to read if we would run all of them with a single command?
```bash
run: pytest modin/pandas/test/dataframe
```

Reply:
This approach allows the developer to quickly identify tests that do not work, as well as estimate the time of individual test cases:
![image](https://user-images.githubusercontent.com/45976948/92384608-0e066d00-f119-11ea-825c-48559f760807.png)

Reply:
got it, thanks for explanation